jupyter-docker/Dockerfile

```diff
@@ -79,10 +79,9 @@ RUN echo "deb $DEBIAN_REPO/debian jessie main\ndeb $DEBIAN_REPO/debian-security
  && wget -q https://storage.googleapis.com/hadoop-lib/gcs/gcs-connector-latest-hadoop2.jar -O $SPARK_HOME/gcs-connector.jar \
 
  # Hail
- && mkdir $HAIL_HOME && git clone https://github.com/broadinstitute/hail.git $HAIL_HOME \
- && cd $HAIL_HOME && git checkout $HAILHASH \
- && wget http://storage.googleapis.com/hail-common/$HAILJAR \
- && wget http://storage.googleapis.com/hail-common/$HAILZIP \
+ && mkdir $HAIL_HOME && cd $HAIL_HOME \
+ && wget http://storage.googleapis.com/hail-common/builds/0.1/jars/$HAILJAR \
+ && wget http://storage.googleapis.com/hail-common/distributions/0.1/$HAILZIP \
```

Comment:
An attempt at optimization. I can't see where we use anything in the git repo. I also don't think we're using anything in the zip either (we're certainly never unzipping it). AFAICT It's just the jar that gets passed around.

Reply:
The zip file is used in spark-defaults (which I found-and-replaced already, but didn't realise). Thanks Joel.